compiler/optimizations/forallOptimizations.cpp

```diff
@@ -88,11 +95,24 @@ static CallExpr *revertAccess(CallExpr *call);
 static CallExpr *confirmAccess(CallExpr *call);
 
 static void symbolicFastFollowerAnalysis(ForallStmt *forall);
+//
+// maps a potential local access to an AggregationCandidate
+std::map<CallExpr *, AggregationCandidateInfo *> preNormalizeAggCandidate;
```

Comment:
It looks to me like this map is only inserted to. Am I missing something?

Reply:
You're right. Will remove.

Reply:
(It appears that I forgot that one in, sorry)